### PR TITLE
Added hackathon timeline to events page

### PR DESCRIPTION
**Pull request by hannagracec**
Used conditional styling to switch between left and right justified for each hackathon. Followed Figma for desktop and mobile view styling.

Desktop View:
image

Mobile View:
image

### CODE REVIEW

File: components/HackathonTimeline.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+const HACKATHON_TIMELINE_CONTENT = 'max-w-[500px] w-[80vw] md:max-w-[1100px]';
+const HACKATHON_TIMELINE_HEADER = 'md:justify-start md:my-[60px] my-[30px] flex justify-center';
+const HACKATHON_EVENT_INFO = 'flex text-center text-xl font-bold mb-[70px] md:items-center flex-col items-center';
+const HACKATHON_TITLE_YEAR = 'text-5xl pb-[15px] italic md:text-4xl';
+const HACKATHON_IMAGE = 'object-cover w-[80vw] md:rounded-xl md:w-[450px] md:h-[250px] md:shadow-[15px_15px_0px_0px_#00D3A9]';
+const HACKATHON_TEXT = 'flex-col md:w-[400px] md:mx-[40px] leading-[1] text-[20px]';
+const HACKATHON_TOPIC_TEXT = 'flex text-left my-[15px] text-[#FF4D6F] md:text-[#FF6B54] md:my-1 md:font-extrabold md:text-[25px]';
+
+const HackathonSection = ({ hackathon, index }) => {
+  let sectionInfoStyling = '';
```
Comment (justin-phxm) [inline components/HackathonTimeline.jsx]: image
Consider this coding style.

**Review by justin-phxm — COMMENTED** [review]
Looks pretty good to me! Just one small suggestion.

File: components/HackathonTimeline.jsx (hunk 1 above)
Comment (hannagracec) [reply components/HackathonTimeline.jsx]: Oh yea, that's much more concise, thank you!

**Review by justin-phxm — DISMISSED** [review]
LGTM!

File: pages/events/index.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import EventsPlaceholder from '../../components/EventsPlaceholder';
+import HackathonTimeline from '../../components/HackathonTimeline';
 
 const Events = () => {
   const placeholder = 'Events page';
   return (
     <>
+      <HackathonTimeline />
       <EventsPlaceholder placeholder={placeholder} />
```
Comment (topanb11) [inline pages/events/index.js]: we can remove this `EventsPlaceHolder` now since the page isn't empty!

**Review by topanb11 — COMMENTED** [review]
thanks Hanna! Just one small comment, rest LGTM!

**Review by nataliey16 — COMMENTED** [review]
Looks good to me! :)

**Review by topanb11 — APPROVED** [review]
thanks Hanna! Code LGTM!

**Pull request merged**